src/main/java/wolox/training/mappers/BookMapper.java

```diff
@@ -0,0 +1,45 @@
+package wolox.training.mappers;
+
+import org.mapstruct.AfterMapping;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.ReportingPolicy;
+import wolox.training.client.models.BookInfoDto;
+import wolox.training.models.Book;
+
+@Mapper(unmappedTargetPolicy = ReportingPolicy.IGNORE, componentModel = "spring")
+public interface BookMapper {
+
+    /**
+     * Method that convert bookInfoDto to Book entity
+     *
+     * @param isbn:        Isbn of the book (String)
+     * @param bookInfoDto: OpenLibrary response ({@link BookInfoDto})
+     * @return {@link Book}
+     */
+    @Mapping(target = "isbn", source = "isbn")
+    @Mapping(target = "subtitle", source = "bookInfoDto.title")
```

Comment:
The API returns you an object with a title field and a subtitle field but here you are mapping the title field to the subtitle :( .